### ISSUE TITLE
ESRGAN-YUV Training 

### ISSUE THREAD

**Issue by Hyunmin-jasper-Cho** [GCU-Graduate-Project-Sharpic/VideoSR#1]
## Things currently done
- [x] **Prepared** Real-world YCbCr Datasets (Flickr2K + BVI-DVC YCbCr converted sequence) 
        (**3850** of training(2450 Flickr2K YCbCr converted + 1400 BVI-DVC) and **200** of validation set)
- [X] **Training** ESRGAN (in_ch / out_ch: 1 | load dataset as one channel using openCV) using YCbCr dataset. 
        (ETA **20H** in colab pro condition | iter: 100'000, epochs: 41)
        (08.11 20:20, iter: 11'500) - current PSNR: 27.53

## Things to do 
- [x] Use ESRGAN-YUV PyTorch trained model as a base generator model of **RealSR**, and training **RealSR** using YCbCr datasets.
